src/cpp/session/modules/SessionPythonEnvironments.R

```diff
@@ -144,6 +147,59 @@
    if (file.exists(prefsPython))
       return(path.expand(prefsPython))
    
+   # on Windows, help users find a default version of Python if possible
```

Comment:
L150-202 were deleted in the short-term fix, here they are restored, but the output gets set to RETICULATE_PYTHON_FALLBACK instead